blog/dev_hackathon_v4_release.md

```diff
@@ -0,0 +1,134 @@
+(sprint_testimonial)=
```

Comment:
This can't be the same one we used for the sprint testimonial. This defines a target so we can then use `` {ref}`target` `` to generate a link that points to this page. It needs to be unique and specific to this file.

Reply:
Updated target